website/docs/developer-docs/docs/style-guide.mdx

```diff
@@ -270,60 +297,126 @@ import Tabs from "@theme/Tabs";
 </Tabs>;
 ```
 
-Tabs improve readability when presenting different configurations or setup options.
+---
 
 ## Error message formatting and troubleshooting
 
-When documenting error messages:
+When documenting errors, follow this structure:
 
-- Display the error message
-- Explain possible causes
-- Offer solutions
+1. **Error Message**: Display the error in a code block.
+2. **Possible Causes**: List common reasons for the error.
+3. **Solutions**: Provide step-by-step fixes or a work-around if there is one.
 
-Example:
+**Example**:
 
-- **Error message:**
+- **Error message**:
 
     ```sh
     Error: Authentication failed. Invalid credentials.
     ```
 
-- **Possible causes:**
+- **Possible causes**:
 
-    - Incorrect username or password
-    - Account is locked due to too many failed login attempts
+    - Incorrect username or password.
+    - Account locked due to multiple failed attempts.
 
-- **Solution:**
+- **Solutions**:
+    1. Verify your credentials.
```

Comment:
We want to use bullets here, not numbers, because numbers imply that you need to follow the thee steps in order. In reality, it might be just one of these things...